driver-core/src/main/java/com/datastax/driver/core/ControlConnection.java

```diff
@@ -594,20 +614,26 @@ private static void updateInfo(
     // After CASSANDRA-9603 (2.0.17, 2.1.8, 2.2.0 rc2) local row contains one more column:
     // - listen_address
```

Comment:
Maybe update this kind of comments to explain the new rows in `peers_v2` (or remove them / have a single comment detailing the table formats in different versions?).